charon/storage.py

```diff
@@ -13,19 +13,20 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-from boto3_type_annotations.s3.service_resource import Object
-from charon.utils.files import read_sha1
-from charon.constants import PROD_INFO_SUFFIX
+import logging
+import mimetypes
+import os
+from typing import Callable, Dict, List, Optional, Tuple
 
 from boto3 import session
+from boto3_type_annotations import s3
+from boto3_type_annotations.s3.service_resource import Object
+from botocore.config import Config
 from botocore.errorfactory import ClientError
 from botocore.exceptions import HTTPClientError
-from botocore.config import Config
-from boto3_type_annotations import s3
-from typing import Callable, Dict, List, Optional, Tuple
-import os
-import logging
-import mimetypes
+
+from charon.constants import PROD_INFO_SUFFIX
+from charon.utils.files import read_sha1
```

Comment:
nitpick: are these importing changing really needed?

Reply:
@ligangty This is auto-format when I do the code formatting based on the PEP 8 style, if you have a preferred unified style setting of imports, welcome to share with me. Thanks.